mantidproject/mantid
# Fix E305 flake8 warnings

Fixes these warnings http://builds.mantidproject.org/job/master_flake8_python3/564/warnings6Result/type.2106131/

These warnings only appear on newer versions of flake8 than is run for PR tests.

**To test:**
Code review, or run flake8 and see that the warnings have gone.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
